ci-operator/step-registry/agent-qe/baremetal/install/ove/disconnected/configure-lso/agent-qe-baremetal-install-ove-disconnected-configure-lso-commands.sh

```
@@ -0,0 +1,174 @@
+#!/bin/bash
+set -euo pipefail
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)